tests/store/artifact/test_databricks_artifact_repo.py

```diff
@@ -274,6 +275,89 @@ def test_log_artifact_azure_blob_client_sas_error(self, databricks_artifact_repo
                 databricks_artifact_repo.log_artifact(test_file.strpath)
             write_credential_infos_mock.assert_called_with(run_id=MOCK_RUN_ID, paths=ANY)
 
+    @pytest.mark.parametrize(
+        ("artifact_path", "expected_location"),
+        [(None, "test.txt"), ("output", "output/test.txt"), ("", "test.txt")],
+    )
+    def test_log_artifact_adl_gen2(
```

Comment:
We should also test that the file chunking is behaving correctly and that the proper number of PATCH's are called when we have to chunk.

Reply:
Updated